Resources/Maps/_Sunrise/planet_prison.yml

```
@@ -83720,6 +83606,11 @@
       rot: -1.5707963267948966 rad
       pos: -24.5,-12.5
       parent: 2
+  - uid: 2331
+    components:
+    - type: Transform
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)